pkg/controller/operatingsystemconfig/actuator.go

```diff
@@ -156,9 +157,16 @@ func (a *actuator) handleReconcileOSC(_ *extensionsv1alpha1.OperatingSystemConfi
 		extensionsv1alpha1.Unit{Name: "locksmithd.service", Command: ptr.To(extensionsv1alpha1.CommandStop), Enable: ptr.To(false)},
 	)
 
-	var err error
-	if extensionUnits, extensionFiles, err = a.configureNTPDaemon(extensionUnits, extensionFiles); err != nil {
-		return nil, nil, fmt.Errorf("error configuring NTP Daemon: %v", err)
+	if a.extensionConfig.NTP.Daemon != configv1alpha1.None {
+		if extensionUnits, extensionFiles, err = a.configureNTPDaemon(extensionUnits, extensionFiles); err != nil {
+			return nil, nil, fmt.Errorf("error configuring NTP Daemon: %v", err)
+		}
+	} else {
+		// Make sure that both services are off
+		extensionUnits = append(extensionUnits,
+			extensionsv1alpha1.Unit{Name: "systemd-timesyncd.service", Command: ptr.To(extensionsv1alpha1.CommandStop), Enable: ptr.To(false)},
+			extensionsv1alpha1.Unit{Name: "ntpd.service", Command: ptr.To(extensionsv1alpha1.CommandStop), Enable: ptr.To(true)},
```

Comment:
Why is Enable set to `false` in the first case and set to `true` in the second one?

Reply:
oops